firelyserver/releasenotes/releasenotes.rst

```diff
@@ -13,10 +13,91 @@ Release notes Firely Server
 .. _upgrade_vonk:
 
 Upgrading Firely Server
---------------
+-----------------------
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Firely Server.
 
+.. _vonk_releasenotes_400:
+
+Release 4.0.0
+-------------
+
+This major version introduces a new name: **Firely Server** instead of Vonk. Other than that, this release contains some significant code changes, which could impact you if you run Firely Server with your own plugins.
+
+Features
+^^^^^^^^
+
+#. Name change Vonk -> Firely Server:
+   #. The main entry point dll (formerly: ``Vonk.Server.dll``) and executable (formerly: ``Vonk.Server.exe``) names have been changed to ``Firely.Server.dll`` and ``Firely.Server.exe`` respectively.
+   #. The name was changed in the CapabilityStatement.name.
+   #. The name of the download zip (from Simplifier) has changed from `vonk_distribution.zip` to `firely-server-latest.zip`. Likewise the versioned zip files have changed as well.
```

Comment:
That does not seem to be the case: https://downloads.simplifier.net/vonk/versions/